packages/flutter/lib/src/material/scrollbar.dart

```diff
@@ -348,7 +348,8 @@ class _MaterialScrollbarState extends RawScrollbarState<_MaterialScrollbar> {
       ..mainAxisMargin = _scrollbarTheme.mainAxisMargin ?? 0.0
       ..minLength = _scrollbarTheme.minThumbLength ?? _kScrollbarMinLength
       ..padding = MediaQuery.of(context).padding
-      ..scrollbarOrientation = widget.scrollbarOrientation;
+      ..scrollbarOrientation = widget.scrollbarOrientation
+      ..ignorePointer = !enableGestures;
```

Comment:
Instead of adding more parameters to the painter too, it may be easier to just pass `enableGestures` to the hit testing methods.

Reply:
It will be better if we can documentation this behavior.